api/handlers/pokemon.go

```diff
@@ -4,46 +4,70 @@ import (
 	"net/http"
 	"strconv"
 
+	"github.com/hoducha/ondemand-go-bootcamp/api/services"
```

Comment:
It was placed outside of the API directory because I think it could be used in another apps like a cronjob or something. But there is no such things in this project so I refactored to move it to the API directory.